src/core/style/mixins.scss

```diff
@@ -251,4 +251,21 @@
     cursor: default;
   }
 
+  &.md-indeterminate ._md-icon{
```

Comment:
The checkbox CSS is used both within the checkbox and the select menu (for multi-select case).